test/framework/src/main/java/org/elasticsearch/cluster/coordination/DeterministicTaskQueue.java

```diff
@@ -309,7 +313,7 @@ public ThreadPoolInfo info() {
 
             @Override
             public Info info(String name) {
-                throw new UnsupportedOperationException();
+                return infos.computeIfAbsent(name, n -> new Info(n, ThreadPoolType.FIXED, random.nextInt(10) + 1));
```

Comment:
Motivation for the map complication here: by using a random size for this pool here we still get the same kind of upload-ordering coverage we had before in the `SnapshotResiliencyTests`